etc/freebsd-rc/syncthing

```diff
@@ -0,0 +1,56 @@
+#!/bin/sh
+#
+#
+# PROVIDE: syncthing
+# REQUIRE: DAEMON
+# KEYWORD: shutdown
+#
+# Add the following lines to /etc/rc.conf to enable this service:
+#
+# syncthing_enable:       Set to NO by default. Set it to YES to enable it.
+# syncthing_home:         Directory where syncthing configuration
+#                         data is stored.
+#                         Default: /usr/local/etc/syncthing
+# syncthing_log_file:     Syncthing log file
+#                         Default: /var/log/syncthing.log
+# syncthing_user:         The user account syncthing daemon runs as what
+#                         you want it to be.
+#                         Default: syncthing
+# syncthing_group:        The group account syncthing daemon runs as what
+#                         you want it to be.
+#                         Default: syncthing
+
+. /etc/rc.subr
+
+name=syncthing
+rcvar=syncthing_enable
+
+start_cmd="${name}_start"
+
+load_rc_config $name
+
+: ${syncthing_enable:=NO}
+: ${syncthing_home=/usr/local/etc/syncthing}
+: ${syncthing_log_file=/var/log/syncthing.log}
+: ${syncthing_user:=syncthing}
+syncthing_group=${syncthing_group:-$syncthing_user}
+
+
+command=/usr/local/bin/syncthing
+pidfile=/var/run/syncthing.pid
+syncthing_flags=" \
+    ${syncthing_home:+-home=${syncthing_home}} \
+    ${syncthing_log_file:+--logfile=${syncthing_log_file}}"
```

Comment:
single slash on one and double on the other - although it doesn't matter which one it is I think it should be consistent

Reply:
Yep, I'll fix it.